static/templates/user_info_popover_content.handlebars

```diff
@@ -100,7 +100,7 @@
     {{#unless is_me}}
     <li>
         <a href="#" class="mention_user">
-            <i class="fa fa-at" aria-hidden="true"></i> {{#tr this}}Reply mentioning user{{/tr}} {{#if is_sender_popover}}<span class="hotkey-hint">(@)</span>{{/if}}
+            <i class="fa fa-at" aria-hidden="true"></i> {{#tr this}}Mention user{{/tr}} {{#if is_sender_popover}}<span class="hotkey-hint">(@)</span>{{/if}}
```

Comment:
templates/zerver/help/mention-a-user-or-group.md needs to change if we're editing this.